GitHub Is An Arse

```diff
@@ -0,0 +1,35 @@
+// Dylan Tkach
+// CPT-263-27
+// Assignment 3 - Loops
+
+package assignment.pkg3.loops;
+
+import java.util.Scanner;
+
+public class Assignment3Loops {
+    
+    public static void main(String[] args) {
+        
+        
+        Scanner keyboard = new Scanner(System.in);
+        print("Please enter a word: ");
+        String word = keyboard.next();
+        
+        while(!word.equalsIgnoreCase("EXIT")){
+        String reversedWord = "";
+        for(int x = word.length() - 1; x >= 0; x--){
+            char value = word.charAt(x);
+            reversedWord += String.valueOf(value);
+            }
+        
```

Comment:
This looks great.  You used the valueOf as Paul suggested.  The only thing that I could see is that it might be helpful to add some more comments.